### PR TITLE
Updates to API Explorer and User pages

### PR DESCRIPTION
**Pull request by RoAnnC1**
Update API Explorer with current info.
Clarify note on User page, minor updates to spacing, heading title caps, etc.

### CODE REVIEW

File: src/api-reference/user/index.markdown [hunk 1]
```diff
@@ -27,8 +25,7 @@ This resource allows you to get profile information for a given user. If a reque
 
 Name | Type | Format | Description
 -----|------| ------ | --------------
-`loginID`|`string` | | The URL-encoded Concur login of the user. Optional.
-
+`loginID`|`string` | | The URL-encoded SAP Concur login of the user. Optional.
```
Comment (charlieconcur) [inline src/api-reference/user/index.markdown]: We need to clean up whitespace and make sure to denote empty cells intentionally with `|-|`.

**Review by charlieconcur — CHANGES_REQUESTED** [review]

**Comment by RoAnnC1** [comment]
Fixed

File: src/api-explorer/index.markdown [hunk 2]
```diff
@@ -3,11 +3,18 @@ title: Swagger API Explorer
...
 
-Browse swagger documents for v3.0 and 3.1 APIs, such as what APIs are available, the resource and method details for each API.
+## Base URIs
 
-## Base URLs
-* **US:** https://www.concursolutions.com/api/{version}
-* **EMEA:** https://eu1.concursolutions.com/api/{version}
+For APIs using the new OAuth2 framework:
 
+* **US**: https://us.api.concursolutions.com/api/{version}
```
Comment (sarraloew) [inline src/api-explorer/index.markdown]: These URLs should not end with /api/{version} as the new v4 URLs don't use that format.

**Review by sarraloew — CHANGES_REQUESTED** [review]

**Review by charlieconcur — APPROVED** [review]

File: src/api-reference/user/index.markdown (hunk 1 above)
Comment (charlieconcur) [reply src/api-reference/user/index.markdown]: Excellent, thanks.

**Comment by sarraloew** [comment]
:shipit:

**Review by sarraloew — APPROVED** [review]

**Pull request merged**